internal/params/config.go

```diff
@@ -90,7 +90,7 @@ var (
 		StakingEpoch:     big.NewInt(2),
 		PreStakingEpoch:  big.NewInt(1),
 		QuickUnlockEpoch: big.NewInt(0),
-		FiveSecondsEpoch: big.NewInt(0),
+		FiveSecondsEpoch: big.NewInt(17700),
```

Comment:
why set to 17700?

Reply:
This is a fork being tested on stressnet. You are right. Change back to 0 since it seems to be never used again.